File: client/src/app/_models/hr/intervwCandAttachment.ts

export interface IIntervwCandAttachment {
     id: number;
     candidateId: number;
     attachmentSizeInBytes: number;
     url: string;
     fileName: string;
     dateUploaded: Date;
}

export class IntervwCandAttachment implements IIntervwCandAttachment {
     id=0;
     candidateId=0;
     attachmentSizeInBytes = 0;
     fileName='';
     url = '';
     dateUploaded = new Date();
     
}

File: api/Entities/HR/IntervwCandAttachment.cs
namespace api.Entities.HR
{
    public class IntervwCandAttachment: BaseEntity
    {
        public int IntervwItemCandidateId { get; set; }
        public int CandidateId { get; set; }
        public int AttachmentSizeInBytes { get; set; }
        public string Url { get; set; }
        public string FileName { get; set; }
        public DateTime DateUploaded { get; set; }
    }
}